File: Modul 6/SOAL 5/PYTHON/PRAK605.py
ordo = int(input())

matriks_A = []
matriks_B = []

print("Matriks A")
for baris in range(ordo):
    elemen1 = list(map(int, input().split()))
    matriks_A.append(elemen1)

print("Matriks B")
for baris in range(ordo):
    elemen2 = list(map(int, input().split()))
    matriks_B.append(elemen2)

print("Matriks AxB")
for baris in range(ordo):
    for kolom in range(ordo):
        hasil = 0
        for x in range(ordo):
            hasil = hasil + matriks_A[baris][x] * matriks_B[x][kolom]
        print(hasil, end=" ")
    print()

File: Modul 6/SOAL 5/C/PRAK605.c
#include <stdio.h>

int main() {
    int ordo, baris, kolom, x, elemen1, elemen2, hasil;
    
    scanf("%d", &ordo);

    int matriks_A[ordo][ordo], matriks_B[ordo][ordo];

    printf("Matriks A\n");
    for(baris = 0; baris < ordo; baris++) {
        for(kolom = 0; kolom < ordo; kolom++) {
            scanf("%d", &elemen1);
            matriks_A[baris][kolom]= elemen1;
        }
    }
    printf("Matriks B\n");
    for(baris = 0; baris < ordo; baris++) {
        for(kolom = 0; kolom < ordo; kolom++) {
            scanf("%d", &elemen2);
            matriks_B[baris][kolom] = elemen2;
        }
    }
    printf("\nMatriks AxB\n");
    for(baris = 0; baris < ordo; baris++) {
        for(kolom = 0; kolom < ordo; kolom++) {
            for(x = 0, hasil = 0; x < ordo; x++) {
                hasil = hasil + matriks_A[baris][x] * matriks_B[x][kolom];
            }
                printf("%d ", hasil);
            }
            printf("\n");
        }
    return 0;
}